tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -39,7 +39,11 @@ function get_container_runtime() {
                 die "invalid node name"
 	fi
 	if echo "$runtime" | grep -qE 'containerd.*-k3s'; then
-		if systemctl is-active --quiet k3s-agent; then
+		if systemctl is-active --quiet rke2-agent; then
+			echo "rke2-agent"
+		elif systemctl is-active --quiet rke2-server; then
```

Comment:
Is there any possibility of having both agent and server on the same node (for testing?)